dsa-23au/java-dsa/sorting/src/main/java/dev/codewithfriends/MergeSort.java

```diff
@@ -0,0 +1,49 @@
+package dev.codewithfriends;
+
+public class MergeSort {
+
+    public static <AnyType extends Comparable<? super AnyType>> void main(String[] args) {
+        mergeSort(args);
+    }
+
```

Comment:
These lines of code are very helpful for me because for me because allow to implement a sorting method no matter the data type.